static_src/app.jsx

```diff
@@ -53,22 +56,29 @@ export default class App extends React.Component {
           </a>
         </div>
       </nav>
-      <div className="container-fluid">
-        <div className="row">
-          <nav className={ sidebarClasses }>
-            { sidebar }
-          </nav>
-          <main className={ mainClasses }>
-            { content }
-          </main>
+      <div className={ titleBarStyles['title_bar'] }>
+        <div class="nav_toggle">
```

Comment:
I think we want them so I'll change the attribute to `className` as React expects